src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp

```diff
@@ -684,12 +1158,114 @@ class spell_paletress_summon_memory : public SpellScriptLoader
         }
 };
 
+class spell_paletress_reflective_shield : public SpellScriptLoader
+{
+    public:
+        spell_paletress_reflective_shield() : SpellScriptLoader("spell_paletress_reflective_shield") { }
+
+        class spell_paletress_reflective_shield_AuraScript : public AuraScript
+        {
+            PrepareAuraScript(spell_paletress_reflective_shield_AuraScript);
+
+            bool Validate(SpellInfo const* /*spellInfo*/) override
+            {
+                if (!sSpellMgr->GetSpellInfo(SPELL_SHIELD_REFLECT))
+                    return false;
+                return true;
+            }
+
+            void HandleScript(AuraEffect* aurEff, DamageInfo& dmgInfo, uint32& absorbAmount)
```

Comment:
Travis (travis-ci) complains here because the parameter `aurEff` is not used in the function directly below:
```cpp
src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp:1177:43:
 fatal error: unused parameter 'aurEff' [-Wunused-parameter]
            void HandleScript(AuraEffect* aurEff, DamageInfo& dmgInfo, uint32& absorbAmount)
                                          ^
```
You can fix it by adding comment separators around the parameter like this: `/*aurEff*/`

(BTW, travis-ci stops looking for these when it has found one, so there could be more further down.)

Reply:
Thank you - i noticed this already. im doing some more changes and commiting more updates soon